src/Components/Timer/Timer.js

```diff
@@ -4,87 +4,87 @@ import style from './Timer.module.scss';
 import { SettingsContext } from '../../Context/SettingsContext'
 import { ViewContext } from '../../Context/ViewContext'
 
-// const COLOR_CODES = {
-//     info: {
-//         color: '#F4D2D0',
-//     }
-// }
-
-// let remainingPathColor = COLOR_CODES.info.color
-
 const CountdownTimer = () => {
-    const [ settings ] = useContext(SettingsContext)
-    const [view, setView] = useContext(ViewContext)
-    // const calculateTimeFraction = () => {
-    //     let timeLeft = this.context.value.m * 60 + this.context.value * 1000
-    //     return timeLeft / props.initTime
-    // }
+  const [settings] = useContext(SettingsContext)
+  const [view, setView] = useContext(ViewContext)
 
-    // const setCircleDashArray =() => {
-    //     const circleDashArray = `${(
-    //         calculateTimeFraction() * 283).toFixed(0)} 283`;
-    //         document.getElementById('base-timer-path-remaining').setAttribute('stroke-dasharray', circleDashArray)
-    // }
+  const timerDone = () => {
+    setView('content-selection')
+  }
 
-    const timerDone = () => {
-        setView('content-selection')
-    }
+  const countdownColorChange = (color) => {
```

Comment:
Thanks for including the screencast in the PR. I love how the circle is formed as the interval counts down!